spec/models/vm_scan/dispatcher_get_eligible_proxies_for_job_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe "JobProxyDispatcherGetEligibleProxiesForJob" do
+RSpec.describe "VmScan::DispatcherGetEligibleProxiesForJob" do
```

Comment:
```suggestion
RSpec.describe "VmScan::Dispatcher#get_eligible_proxies_for_job" do
```

Reply:
Not necessary for this PR, but these describes are really strange...it's like an entire rspec for a single method :)

Reply:
Yeah I agree, I didn't want to be changing to much so I left these as is, but the feel like they should just be another context in the `vm_scan::dispatcher` spec